src/rfq/protocols/hashflow/client.rs

```diff
@@ -433,35 +431,68 @@ impl RFQClient for HashflowClient {
 
                     let mut quote_attributes: HashMap<String, Bytes> = HashMap::new();
                     quote_attributes.insert("pool".to_string(), quote.quote_data.pool);
+                    if let Some(external_account) = quote.quote_data.external_account {
+                        quote_attributes.insert("external_account".to_string(), external_account);
+                    }
                     quote_attributes.insert("trader".to_string(), quote.quote_data.trader);
+                    if let Some(effective_trader) = quote.quote_data.effective_trader {
+                        quote_attributes.insert("effective_trader".to_string(), effective_trader);
+                    }
+                    quote_attributes.insert("base_token".to_string(), quote.quote_data.base_token);
```

Comment:
aaaahhh I think I understand what you are doing here.. You are adding everything here to make encoding easier.. but we already have this info in the `SignedQuote.base_token`. Do we need to have it in multiple places? I don't have a strong strong preference in this though

Reply:
same for `quote_token`, `base_token_amount` and `quote_token_amount`